build.gradle

```diff
@@ -101,7 +101,7 @@ test {
     reports.html.enabled = true
     testLogging {
         exceptionFormat = 'full'
-        events "passed", "skipped", "failed", "standardOut", "standardError"
+        events "passed", "skipped", "failed"//, "standardOut", "standardError"
```

Comment:
Why not remove?

Reply:
Because during test running for troubleshooting and optimization reasons you need quickly uncomment more verbose output and then back again.